hoomd/md/force.py

```diff
@@ -238,24 +253,18 @@ def update_coeffs(self):
         pass
 
 
-class active(_Force):
+class Active(_Force):
     R""" Active force.
 
     Args:
+        filter (:py:mod:`hoomd.filter`): Subset of particles on which to apply active
+            forces.
         seed (int): required user-specified seed number for random number generator.
-        f_list (list): An array of (x,y,z) tuples for the active force vector for each individual particle.
-        t_list (list): An array of (x,y,z) tuples that indicate active torque vectors for each particle
-        group (:py:mod:`hoomd.group`): Group for which the force will be set
-        orientation_link (bool): if True then forces and torques are applied in the particle's reference frame. If false, then the box
-         reference frame is used. Only relevant for non-point-like anisotropic particles.
-        orientation_reverse_link (bool): When True, the particle's orientation is set to match the active force vector. Useful for
-         for using a particle's orientation to log the active force vector. Not recommended for anisotropic particles. Quaternion rotation
-         assumes base vector of (0,0,1).
-        rotation_diff (float): rotational diffusion constant, :math:`D_r`, for all particles in the group.
         constraint (:py:class:`hoomd.md.update.constraint_ellipsoid`) specifies a constraint surface, to which particles are confined,
           such as update.constraint_ellipsoid.
+        rotation_diff (float): rotational diffusion constant, :math:`D_r`, for all particles in the group.
 
```

Comment:
Add an `Attributes:` section and document `filter`, `seed`, `rotation_diff`, `constraint`, `active_force` and `active_torque`)